src/cloud/components/DocTagsListItem.tsx

```diff
@@ -91,15 +91,15 @@ const DocTagsListItem = ({
 const Container = styled.div`
   display: inline-flex;
   flex-wrap: wrap;
-  padding: 0.25em 0.5em;
+  padding: 0 0.5em;
   position: relative;
-  margin: 0 ${({ theme }) => theme.sizes.spaces.xsm}px;
+  margin-right: ${({ theme }) => theme.sizes.spaces.xsm}px;
   font-size: ${({ theme }) => theme.sizes.fonts.df}px;
-  border-radius: 3px;
+  border-radius: 4px;
```

Comment:
Use theme whenever possible to avoid bringing a lot of variables

```
${({ theme }) => theme.borders.radius}px
```

Reply:
Fixed